- description: Query from the remote schema returning errors only
  url: /v1/graphql
  status: 200
  headers:
    x-fake-operation-name: ErrorOnly
  response:
    data:
    errors:
    - extensions:
        code: validation-failed
        path: "$.selectionSet.test_lol"
      message: 'field ''test'' not found in type: ''query_root'''
  query:
    query: |
      query ErrorOnly {
        test {
          id
          created_at
        }
      }
